tools/interactive/interactivetool_ml_jupyter_notebook.xml

```diff
@@ -0,0 +1,94 @@
+<tool id="interactive_tool_ml_jupyter_notebook" tool_type="interactive" name="GPU enabled Interactive Jupyter Notebook for Machine Learning" version="0.1">
+    <requirements>
+        <container type="docker">docker.io/anupkumar/docker-ml-jupyterlab:galaxy-integration</container>
```

Comment:
Can you include a version for the container ?

Reply:
@mvdbeek Thanks for your comment, but I am not sure what should be the version number of a container when the tag name of the container is present. Can you give an example of a version? Thanks!

Reply:
You can add a version at the end of the tag?

Reply:
ok, something like `:tag-name-version`. 
An example could be - `docker.io/anupkumar/docker-ml-jupyterlab:galaxy-integration-0.1`?

Reply:
sounds good